bodhi/server/templates/update.html

```diff
@@ -575,10 +575,12 @@ <h1>
       % endif
 
     </h1>
-    <h4>
+    <h4 id='shortdesc'>
       ${update.type} update
-      in ${update.release.long_name}
-      for ${update.beautify_title(amp=True) | n}
+      in ${update.release.long_name} for
+      % for build in update.builds:
+        <a href="https://apps.fedoraproject.org/packages/{$build.package.name}">${build.package.name}</a>${'' if loop.last else ','}
```

Comment:
We don't want to hardcode any URLs into Bodhi - this will cause issues for deployments that are not in Fedora, and will also be incorrect for our staging deployment.

Reply:
It is an external link. Where do you propose to store it?

Reply:
We should put it into Bodhi's settings. To do this, you add the setting to ```production.ini```, with the default commented out and some text over it explaining what the setting does. There is a proposal from @cverna to remove the Packages app, so I think we probably want the default to be the empty string and have this code only show the link if the setting is truthy.

The second place the setting needs to be added is ```bodhi.server.config```, and this is where the default is defined.

Reply:
No time for investigation. How to reference config from template properly?

Reply:
Here's an example:

https://github.com/fedora-infra/bodhi/blob/3.13.3/bodhi/server/templates/update.html#L54